tests/CMakeLists.txt

```diff
@@ -55,15 +55,12 @@ if(BUILD_AFQMC)
   endif()
 endif()
 
-if(HAVE_PYSCF)
-  if(NOT QMC_COMPLEX)
-    include("${qmcpack_SOURCE_DIR}/CMake/python.cmake")
-    check_python_reqs("numpy;h5py;pyscf;PyscfToQmcpack;PyscfToQmcpack_Spline" pyscf_workflow add_tests)
-
-    if(add_tests)
-      message("Python dependencies met. Adding PySCF workflow tests")
-      add_subdirectory("pyscf")
-    endif()
+if(NOT QMC_COMPLEX)
+  include("${qmcpack_SOURCE_DIR}/CMake/python.cmake")
+  check_python_reqs("numpy;h5py;pyscf;PyscfToQmcpack;PyscfToQmcpack_Spline" pyscf_workflow add_tests)
+  if(add_tests)
+    message("Python dependencies met. Adding PySCF workflow tests")
+    add_subdirectory("pyscf")
   endif()
```

Comment:
Please add a message saying "Python dependencies not met. Skipping PySCF workflow tests"